test/extended/networking.sh

```diff
@@ -24,7 +24,6 @@ NETWORKING_E2E_MINIMAL="${NETWORKING_E2E_MINIMAL:-}"
 
 DEFAULT_SKIP_LIST=(
   # TODO(marun) This should work with docker >= 1.10
-  "openshift router"
   "\[Feature:Federation\]"
```

Comment:
So the "#TODO" comment apparently applies only to the router tests, so if you're removing the router tests from the skip list, you need to remove the comment as well